### DISCUSSION TITLE
How to get a schedule view of just today for certain calendars

### DISCUSSION THREAD

**Discussion by hunterwebsiteservices** [YukiGasai/obsidian-google-calendar#162] [Q&A]
I am struggling to get what I would like

I am using an Obsidian note to pull in Todoist and Google Calendar. And I'd like to put headings in the note for Today, Tomorrow, etc for the next 7 days. With under Today for example. Todoist top priority stuff and Google Calendar events.

So I would like to be able to do something like:

```gEvent
type: schedule
date: today
include: ['Calendar 1', 'Calendar 1']
```

```gEvent
type: schedule
date: tomorrow
include: ['Calendar 1', 'Calendar 1']
```

```gEvent
type: schedule
date: +2days
include: ['Calendar 1', 'Calendar 1']
```

I am unable to get it to work with all 3 values. `type: schedule` works in combination with `[include](include: [''Calendar 1', 'Calendar 1'])` but as soon as `date: today` is added it breaks. What is the correct syntax for that? And how to do this for days after "tomorrow"?

**Comment by YukiGasai** [comment]
I am not quite sure what you want to achieve.
My guess is, that you want to display the schedule view for today, tomorrow and the next 5 days of the week?
If you want to display them all in one block, you can use the option `timespan` to set the amount of days.
If you want them separated from each other, you can use the following text:

\# Today
~~~
```gEvent
type: schedule
date: +0days
include: ['Test', 'Leben']
timespan: 1
```
~~~
\# Tomorrow
~~~
```gEvent
type: schedule
date: +1days
include: ['Test', 'Leben']
timespan: 1
```
~~~
\# +2
~~~
```gEvent
type: schedule
date: +2days
include: ['Test', 'Leben']
timespan: 1
```
~~~
\# +3
~~~
```gEvent
type: schedule
date: +3days
include: ['Test', 'Leben']
timespan: 1
```
~~~
\# +4
~~~
```gEvent
type: schedule
date: +4days
include: ['Test', 'Leben']
timespan: 1
```
~~~
\# +5
~~~
```gEvent
type: schedule
date: +5days
include: ['Test', 'Leben']
timespan: 1
```
~~~
\# +6
~~~
```gEvent
type: schedule
date: +6days
include: ['Test', 'Leben']
timespan: 1
```
~~~

**Comment by hunterwebsiteservices** [comment]
Thank you for the response. I have tried the above and it is not working. Is it possibly a bug?

```gEvent
type: schedule
date: +0days
timespan: 1
include: ['Calendar 1', 'Calendar 2']
```

Only thing that words for me is

```gEvent
type: schedule
```

Adding any of the other fields breaks it. I believe I had `day` view working with the `include` parameter.
